AirCasting/RootAppView.swift

```diff
@@ -87,19 +89,31 @@ struct MainAppView: View {
     @EnvironmentObject private var urlProvider: UserDefaultsBaseURLProvider
     @EnvironmentObject private var userAuthenticationSession: UserAuthenticationSession
     @EnvironmentObject private var bluetoothManager: BluetoothManager
+    @EnvironmentObject private var user: UserState
+    
+    @State private var isUserLoggingOut: Bool = false
```

Comment:
do we need this? Can't we just use `UserState`: `LoadingView(isShowing: $user.isLoggingOut...)` ?